GameData/RealismOverhaul/RO_SAFConfig.cfg

```diff
@@ -0,0 +1,90 @@
+/// Example config - Replace 'MyPartWithSAFModule' with your part name
+/// @PART[MyPartWithSAFModule]:FOR[RealismOverhaul]
+/// {
+/// 	@MODULE[ModuleSimpleAdjustableFairing]
+/// 	{
+/// 		ROSAFRescale = 1.81 //Make everything 1.81 times bigger
+/// 	}
+/// }
+
+@PART[*]:HAS[@MODULE[ModuleSimpleAdjustableFairing]:HAS[#ROSAFRescale]]:AFTER[RealismOverhaul]
+{
+	@MODULE[ModuleSimpleAdjustableFairing]
+	{
+		&scale = 1
+		@scale *= #$ROSAFRescale$
+		@segmentLength *= #$ROSAFRescale$
+		@shieldingCenter[*] *= #$ROSAFRescale$
+		@shieldingBaseRadius *= #$ROSAFRescale$
+		@editorOpenOffset[*] *= #$ROSAFRescale$
+
+		@WALL_BASE
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@WALL
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@CONE
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@CAP
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
```

Comment:
We're sure that all of these things exist? (Though if we aren't it gets a lot more annoying to patch...)

Reply:
If they do not exist, scaling them would mean nothing

Reply:
If they don't exist, using `@` would result in errors.

Reply:
`@` tag fails silently, according to DRVeyl